docs/man_pages/index.md

```diff
@@ -32,6 +32,7 @@ Command | Description
 [platform list](project/configuration/platform.html) | Lists all platforms that the project currently targets.
 [platform&nbsp;remove&nbsp;`<Platform>`](project/configuration/platform-remove.html) | Removes the selected platform from the platforms that the project currently targets. This operation deletes all platform-specific files and subdirectories from your project.
 [platform update `<Platform>`](project/configuration/platform-update.html) | Updates the NativeScript runtime for the specified platform.
+[resources-update](project/configuration/resources-update.html) | Updates the App_Resources/<platform>'s internal folder structure to conform to that of an Android project.
```

Comment:
Just an idea " conform to that of an Android project." might sound more concrete if we change "Android project" to "Android Native project" or "Android Studio project".